CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ## [Unreleased 3.0](https://github.com/opensearch-project/k-NN/compare/2.x...HEAD)
 ### Features
 ### Enhancements
+* Remove FSDirectory dependency and deprecated FileWatcher [#2182](https://github.com/opensearch-project/k-NN/pull/2182)
```

Comment:
Move to 2.x changelog I believe. Also, not sure this is correct if FSDirectory dependency is still there.

Reply:
oh, isn't this change be merged into main branch?
should I put this under both 3.0, 2.x sections? or just putting it under 2.x should be enough?
if putting it under 2.x should be enough then when we add elements under 3.0 section?

Reply:
2.x only if you will backport